src/app/shared/search/search-filters/search-filters.component.ts

```diff
@@ -82,6 +83,10 @@ export class SearchFiltersComponent implements OnInit, OnDestroy {
       return filters;
     }));
     this.searchLink = this.getSearchLink();
+    this.searchConfigService.getConfigurationSearchConfig(this.currentConfiguration).pipe(distinctUntilChanged()).subscribe(searchConfig => {
```

Comment:
@GauravD2t 

Since the searchConfig is no longer needed to the advanced-search.component i think you can remove all this part (and related variables) avoiding to make an additional rest request